Makefile

```diff
@@ -161,6 +161,9 @@ local-image:
 local-image-push:
 	$(IMAGE_PUSH_CMD) $(IMAGE_PUSH_EXTRA_OPTS) $(IMAGE)
 
+replace-nto-image:
+	IMAGE=$(IMAGE) hack/replace-nto.sh
```

Comment:
If possible, let's avoid adding a new Makefile target for this.

1) This will not work as is, you'd have to say something like "make IMAGE=whatever replace-nto-image"
2) I'd say this is overcomplicated and I don't see any value adding a new target for this.  There is plenty of hack/* scripts that don't have a makefile target.